tests/multigrid-global-coarsening/mg_transfer_a_06.cc

```diff
@@ -0,0 +1,326 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2020 - 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+/**
+ * Test prolongation and restriction with uniformly refined meshes.
+ *
+ * To compare with fe/up_and_down_simplex.
```

Comment:
What does the test actually do?

Reply:
I used it as a sanity check to see if the operations done are reasonable, but it does not add any additional value compared to `mg_transfer_a_05` apart from testing the `interpolate()` function. So maybe just remove this?

Reply:
So the test for `interpolate()` is not done elsewhere? In that case, I would suggest to keep that part as a separate test, name the test case accordingly (or write the comment here on what the special purpose of this test is). I like to have tests that are specific, so in case something fails I can look into the test and see immediately what the special purpose of this particular test is.

Reply:
Changed the test to only test the `interpolate()` function and added a comment about it.